omod/src/main/webapp/resources/htmlforms/122a-HIVCare_ARTCard-SummaryPage.xml

```diff
@@ -0,0 +1,1732 @@
+                                <span class="ARTCareEntryPointEnableDisable1">
+                                        <obs conceptId="90200"
+                                             answerConceptIds="90012,90016,99593,99610,90019,90013,90015,90018,165224,90002"
+                                             answerLabels="eMTCT,TB,YCC,SMC,Outreach,Out Patient,STI,Inpatient,KP Clinic,Other"
+                                             required="true"/>
+                                </span>
+                            </div>
+                            <div class="form-group col-2">
+                                <label>Other (Specify)</label>
+                            </div>
+                            <div class="form-group col-2">
+                                <span class="ARTCareEntryPointEnableDisableChild1">
+                                    <obs conceptId="99115" labelText=""/>
+                                </span>
+                            </div>
+                        </obsgroup>
+                    </div>
+                </section>
+                <section>
+                <table>
+                    <div class="row">
+                        <div class="col-2" >
+                            <label>Medicine pick-up if ill:</label>
+                        </div>
+                        <div class="col-2" >
+                            <obs size="15" conceptId="99142"/>
+                        </div>
+                    </div>
+
+                    <div class="row">
+                        <div class="col-2" >
+                            <label>Relationship to index clients</label>
+                        </div>
+                        <div class="col-2" >
+                            <obs conceptId="99725" answerConceptIds="90286,90280,90288,90284,90279,5618,90002"
+                                 answerLabels="Cousin,Child,Partner/Spouse,GrandChild,Parent,Friend,Other"/>
+                        </div>
+                    </div>
+                    <div class="row">
+                        <div class="col-2" >
+                            <label>Telephone</label>
+                        </div>
+                        <div class="col-2" >
+                            <obs conceptId="90278" size="12" class="ugphone" id="90278"/>
+                        </div>
+                        <div class="col-2" >
+                            <label>Whose</label>
+                        </div>
+                        <div class="col-2" >
+                            <obs conceptId="99118"/>
+                        </div>
+                    </div>
+                </table>
+                </section>
+            </div>
+
+            <section id="art-care-cohort-section">
+                <section>
+                    <h3>ART Care Start</h3>
+                    <section>
+                        <table border="0" id="art-care">
+                            <tbody style="display:block;">
```

Comment:
@dbaluku use Bootstrap styles and approaches - see @solemabrothers and @semujju use of cards to have a consistent approach